File: JAva/RangeLister.java
class RangeLister {
    // Método para criar um intervalo de números inteiros
    int[] makeRange(int lower, int upper) {
        int[] range = new int[(upper - lower) + 1];
        
        for (int i = 0; i < range.length; i++) {
            range[i] = lower + i; // Evita modificar diretamente o valor de 'lower'
        }
        return range;        
    }

    public static void main(String[] arguments) {
        RangeLister lister = new RangeLister();
        int[] range = lister.makeRange(1, 10);

        System.out.print("The array: [ ");
        for (int num : range) { // Uso de for-each para simplificar
            System.out.print(num + " ");
        }
        System.out.println("]");
    }
}

File: JAva/RangeLister.py
class RangeLister:
    def make_range(self, lower, upper):
        # Cria uma lista com os números no intervalo inclusivo
        return list(range(lower, upper + 1))

# Método principal
if __name__ == "__main__":
    lister = RangeLister()
    range_list = lister.make_range(1, 10)

    print("The array: [", end=" ")
    for num in range_list:
        print(num, end=" ")
    print("]")